dojo-dungeon/src/dojo/rooms/rooms/riddle/Fragen_Pattern.java

```diff
@@ -30,12 +28,12 @@
  * werden. Die erkannten Design Patterns müssen dann dem Zauberer mitgeteilt werden.
  */
 public class Fragen_Pattern extends Room {
-  private final String FILENAME = "dojo-dungeon/todo-assets/Fragen_Pattern/UML_Klassendiagramm";
-  private final String[] expectedPatterns = {"Observer", "Visitor"};
+  private final String fileNamePrefix =
```

Comment:
konstanten werden allcaps geschrieben?

Reply:
Danke. Das wäre doch eigentlich eine Aufgabe für unser Checkstyle?

Reply:
im prinzip schon. aber `final` ist in java ja nicht unbedingt eine konstante, insofern möchte ich da keine weiteren checkstyleregeln ergänzen.